Source/WebKit/NetworkProcess/storage/CacheStorageManager.h

```diff
@@ -51,7 +46,7 @@ struct CacheStorageRecord;
 struct CacheStorageRecordInformation;
 
 
-class CacheStorageManager : public CanMakeWeakPtr<CacheStorageManager> {
+class CacheStorageManager : public RefCountedAndCanMakeWeakPtr<CacheStorageManager> {
```

Comment:
How is that safe with the `CheckedRef<CacheStorageRegistry> m_registry;` data member?

Reply:
Oh, indeed.. We need to make that one ref counted as well.